tests/test_quantitative_model.py

```
@@ -306,63 +306,141 @@ def test_sample_proba_returns_valid_probabilities_smab(
+    rewards, context, n_features, dimension_and_quantities, monkeymodule
+):
+    dimension, quantities = dimension_and_quantities
+    model = QuantitativeBayesianNeuralNetwork.cold_start(dimension=dimension, n_features=n_features)
+    init_params = deepcopy(model.bnn.model_params)
+    n_model_features = n_features + dimension
+
     monkeymodule.setattr(
         pybandits.model,
         "fit",
-        lambda *args, **kwargs: FakeApproximation(n_features=n_features),
+        lambda *args, **kwargs: FakeApproximation(n_features=n_model_features),
```

Ruff (0.15.2)

[warning] Unused lambda argument: `args`

(ARG005)